NuRadioReco/utilities/signal_processing.py

```diff
@@ -27,3 +31,54 @@ def half_hann_window(length, half_percent=None, hann_window_length=None):
     half_hann_widow[-hann_window_length:] = hann_window[hann_window_length:]
 
     return half_hann_widow
+
+
+def add_cable_delay(station, det, sim_to_data=None, trigger=False, logger=None):
+    """
+    Add or subtract cable delay by modifying the `trace_start_time`.
+
+    Parameters
+    ----------
+    station: Station
+        The station to add the cable delay to.
+
+    det: Detector
+        The detector description
+
+    trigger: bool
+        If True, take the time delay from the trigger channel response.
+        Only possible if ``det`` is of type `rnog_detector.Detector`. (Default: False)
+
+    logger: logging.Logger, default=None
+        If set, use ``logger.debug(..)`` to log the cable delay.
+
+    See Also
+    --------
+    `NuRadioReco.modules.channelAddCableDelay.channelAddCableDelay`
+    """
```

Comment:
I just realized this is just `channelAddCableDelay.run`, with the added option to apply delays to the additional trigger channels only... I guess the code isn't duplicated because you made channelAddCableDelay wrap this function, but I'm not sure I understand why you didn't just add the `trigger` flag to `channelAddCableDelay` so it's all in one place.

I would probably be a happier man if I could understand your motivation for this! But I can live with this, at least the code is in one place and they link to each other.

Reply:
Because I am using this helper function also in the 3 hardwareResponseIncorperator modules. We could call the cable delay adder module within this modules but this would produce quite some overhead (boiler) code which is not needed with this solution.

Reply:
Am I happy? Not that happy. But it's a good enough reason. (It's true a function makes more sense than a class in this case as channelAddCableDelay doesn't use `self` at all... one could make the two completely identical (apart from the unused `event` in the signature) by just adding a `@staticmethod` decorator and using that. But that would not necessarily be a much cleaner solution and I think I really will be able to live with this.)